webpack/assets/javascripts/react_app/components/SettingsTable/SettingsTableReducer.js

```diff
@@ -0,0 +1,18 @@
+import Immutable from 'seamless-immutable';
+
+import { SET_UPDATE_SETTING } from './SettingsTableConstants';
+
+const initialState = Immutable({
+  toEdit: {},
```

Comment:
weird name and weird placement. this is in `SettingTableReducers` right? but it's shared with the update form.
Ideal state would be, in the shared data of all settings table just noting `'editing': <id>` of the setting that should get edited. The Wrapped form would check on that state attribute, and if the id is set, it would get the proper setting from the store.

Reply:
I moved the state, but I kept storing the whole setting - I already have access to the setting object and I know I will need it later, that will save me an additional filtering of settings from store.